apidoc/Titanium/Calendar/Event.yml

```diff
@@ -76,6 +77,8 @@ methods:
         When an event is saved, it is updated in the Calendar database. Any fields you did 
         not modify are updated to reflect the most recent value in the database. If the 
         event has been deleted from the database, it is re-created as a new event.
+
+        On Android, this method only updates the recurrence rule (if set).
```

Comment:
@ypbnv Do we have a parity ticket for this?

Reply:
@hansemannn No, I think. It was added with all other changes for this ticket: https://jira.appcelerator.org/browse/TIMOB-24248